README.md

```diff
@@ -5,6 +5,34 @@
 
 This `ember-cli` addon adds a simple flash message service and component to your app. Just inject with `Ember.inject.service` and you're good to go!
```

Comment:
Should this section go somewhere below the TOC?

Reply:
I was thinking that a short introduction would be nice to have before the table of contents. I made some changes to it by the way, what do you think.
I try to avoid words like 'just', 'simply', 'easily', etc. wdyt?

Reply:
That's much appreciated, thank you!